app/views/unattended/report_templates/host_-_statuses.erb

```diff
@@ -22,9 +22,10 @@ model: ReportTemplate
 <%- report_headers 'Name', 'Global' -%>
 <%- machine_readable = input('Output style') == 'Machine-readable' -%>
 <%- load_hosts(search: input('hosts'), includes: :host_statuses).each_record do |host| -%>
-<%-   report_row({
```

Comment:
To keep the diff smaller, you could use `report_row(**{` as well.

Reply:
I tried that too but I feel like this is more readable.